.zuul.yaml

```diff
@@ -13,16 +15,39 @@
     run: .zuul/playbooks/run.yaml
     nodeset: ubuntu-bionic-large
     vars:
-      go_version: '1.13.15'
+      go_version: '1.15.8'
```

Comment:
golang should be 1.16.

Document was updated ;)
https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md#minimum-go-version

Reply:
Done